src/pyclass.rs

```diff
@@ -165,13 +165,31 @@ where
     // buffer protocol
     type_object.tp_as_buffer = to_ptr(<T as class::buffer::PyBufferProtocolImpl>::tp_as_buffer());
 
+    let (new, call, mut methods, attrs) = py_class_method_defs::<T>();
+
     // normal methods
-    let (new, call, mut methods) = py_class_method_defs::<T>();
     if !methods.is_empty() {
         methods.push(ffi::PyMethodDef_INIT);
         type_object.tp_methods = Box::into_raw(methods.into_boxed_slice()) as *mut _;
     }
 
+    // class attributes
+    if !attrs.is_empty() {
+        unsafe {
+            type_object.tp_dict = ffi::PyDict_New();
```

Comment:
Same as above.
You can use `let dict = pyo3::types::PyDict::new();` and `type_object.tp_dict = dict.to_object(py).into_ptr()`.
The current code is also OK, though.